Missing option within CMake for find_package(Curses REQUIRED)

On some Linux systems an additional option must be set to get the given Code to link correctly.
Default repository clone ran me into the following error:

```
[100%] Linking CXX executable monitor
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: CMakeFiles/monitor.dir/src/ncurses_display.cpp.o: undefined reference to symbol 'cbreak'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: /lib64/libtinfo.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

Checking the CMake documentation [https://cmake.org/cmake/help/v3.13/module/FindCurses.html](https://cmake.org/cmake/help/v3.13/module/FindCurses.html) states: `Set CURSES_NEED_NCURSES to TRUE before the find_package(Curses) call if NCurses functionality is required.`
If i follow that, everything works as expected.

I'll provide a PR for that minor patch

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.6)
 project(monitor)
 
+set(CURSES_NEED_NCURSES TRUE)
 find_package(Curses REQUIRED)
 include_directories(${CURSES_INCLUDE_DIRS})
 

